src/sql/src/statements/show.rs

```diff
@@ -74,18 +155,78 @@ pub struct ShowTables {
     pub full: bool,
 }
 
+impl ShowTables {
+    pub fn kind(&self) -> &ShowKind {
+        &self.kind
+    }
+
+    #[inline]
+    fn format_database(&self) -> String {
+        match &self.database {
+            None => String::default(),
+            Some(database) => {
+                format!("IN {}", database)
+            }
+        }
+    }
+
+    #[inline]
+    fn format_full(&self) -> &str {
+        if self.full {
+            "FULL"
+        } else {
+            ""
+        }
+    }
```

Comment:
Let's try to all inline the function manually, instead of use an extra function. In this way, we can handle whitespace more precise also.

Like sqlparser-rs:

```rust
            Statement::SetTimeZone { local, value } => {
                f.write_str("SET ")?;
                if *local {
                    f.write_str("LOCAL ")?;
                }
                write!(f, "TIME ZONE {value}")
            }
```